packages/core/src/lib/implementation/execute-plugin.ts

```diff
@@ -86,18 +93,28 @@ export async function executePlugin(
       };
     });
 
-    return {
+    const pluginReport: PluginReport = {
       version,
       packageName,
-      slug,
       title,
+      slug,
       icon,
-      description,
-      docsUrl,
       date,
       duration,
       audits,
     };
+
+    if (description) {
+      pluginReport.description = description;
+    }
+    if (docsUrl) {
+      pluginReport.docsUrl = docsUrl;
+    }
+    if (groups) {
+      pluginReport.groups = groups;
+    }
```

Comment:
I'd prefer if we did this in a less imperative way.

```ts
return {
  ...description && { description },
  // ...
}
```

Is it important to omit the falsy properties entirely, though?

Reply:
Done, now its more impereative.

> Is it important to omit the falsy properties entirely, though?

Less data saved/sent maybe. But I have no strong feelings.